File: BEGINNER/1036 Fórmula de Bhaskara.py
a, b, c = map(float, input().split())
delta = (b ** 2) - 4 * (a * c)
if delta <= 0 or a == 0:
    print('Impossivel calcular')
else:
    print(f'R1 = {(((-1) * (b)) + (delta ** 0.5)) / (2 * a):.5f}\n'
          f'R2 = {(((-1) * (b)) - (delta ** 0.5)) / (2 * a):.5f}')

File: C/BEGINNER/1036 Fórmula de Bhaskara.c
#include <stdio.h>
#include <math.h>

int main(int argc, char const *argv[])
{
    double A,B,C, delta;
    scanf("%lf %lf %lf", &A, &B, &C);
    delta = (B*B) - 4 * A * C;
    if (delta < 0 || A == 0)
    {
        printf("Impossivel calcular\n");
    }
    else
    {
    printf("R1 = %.5lf\n", ((-(B) + sqrt(delta)) / (2 * A)));
    printf("R2 = %.5lf\n", ((-(B) - sqrt(delta)) / (2 * A)));
    }
    return 0;
}
